server/src/defaultConfig.json

```diff
@@ -7,5 +7,16 @@
   "colorPalette": "#0078d4",
   "waitingTitle": "Thank you for choosing Lamna Healthcare",
   "waitingSubtitle": "Your clinician is joining the meeting",
-  "logoUrl": ""
+  "logoUrl": "",
+  "postCall": {
+    "survey": {
+      "type": "onequestionpoll",
+      "options": {
+        "title": "Tell us how we did",
+        "prompt": "How satisfied are you with this virtual appointment's audio and video quality?",
+        "pollType": "likeOrDislike",
+        "saveButtonText": "Continue"
+      }
+    }
+  }
```

Comment:
Let's also add the `cosmosDb` property

Reply:
Let's add those default values like in the docs -- "Set the endpoint to authenticate using Managed Identity method" etc etc